mitodl/mit-open
# Stop using name "infinite-corridor". Call frontend "mit-open".

#### What are the relevant tickets?
Closes #55 

#### What's this PR do?
This PR changes the name of the frontend app from `"infinite-corridor"` to `"mit-open"`.

#### How should this be manually tested?
Test development mode:
1. Run `docker compose up`. Visit http://localhost:8063/. All should be well.

Test production mode:
1. Stop your docker containers.
2. Set `NODE_ENV=production` in your `.env` file.
3. `docker compose up`
4. Visit http://localhost:8063/; it should load. Check the network tab to make sure the css/js assets are being loaded from something like 
http://localhost:8063/static/mit-open/style-2bb2e7e98f8c818d.js (and not from the webpack port on 8062).
